README.md

```diff
@@ -327,6 +327,13 @@ zoom.addEventListeners('show', event => {
 })
 ```
 
+## Compatibility
+| IE | Edge | Chrome | Firefox | Safari |
+|----|------|--------|---------|--------|
+| No | 12   | 64     | 58      | 11     |
```

Comment:
A few suggestions:

- IE10+ will be supported once #35 has been merged.
- Edge 12 requires the same `<template>` polyfill as IE10+ for custom template support. If we intend to note this for IE, we should do the same for Edge 12.
- The evergreen browser versions should list the earliest compatible versions.

I did a quick test of the examples in various browsers to determine the earliest compatible version. Taking the polyfill notice into account, an updated table would look more like this:

| IE | Edge | Chrome | Firefox | Safari |
|----|------|--------|---------|--------|
| 10+<sup>1</sup> | 12+<sup>2</sup> | 36     | 34      | 9+     |

<sup>1</sup> IE10+ requires a `<template>` polyfill when using [custom templates](#using-a-custom-template).
<sup>2</sup> Edge 12 requires a `<template>` polyfill when using [custom templates](#using-a-custom-template).